test/send/sendFCM.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-unresolved */
+/* eslint-env mocha */
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { Messaging as fbMessaging } from 'firebase-admin/messaging';
+import PN from '../../src';
+import { testPushSuccess } from '../util';
+
+const method = 'fcm';
+const regIds = [
+  'APA91bFQCD9Ndd8uVggMhj1usfeWsKIfGyBUWMprpZLGciWrMjS-77bIY24IMQNeEHzjidCcddnDxqYo-UEV03xw6ySmtIgQyzTqhSxhPGAi1maf6KDMAQGuUWc6L5Khze8YK9YrL9I_WD1gl49P3f_9hr08ZAS5Tw',
+];
+const message = {
+  title: 'title',
+  body: 'body',
+  sound: 'mySound.aiff',
+  custom: {
+    sender: 'banshi-test',
+  },
+};
+const fcmOpts = {
+  fcm: {
+    name: 'testAppName',
+    credential: { getAccessToken: () => Promise.resolve({}) },
+  },
+  isLegacyGCM: false,
+};
+const pn = new PN(fcmOpts);
+
+const testSuccess = testPushSuccess(method, regIds);
+
+let sendMethod;
+
+function sendOkMethod() {
+  return sinon.stub(
+    fbMessaging.prototype,
+    'sendEachForMulticast',
+    function sendFCM(firebaseMessage) {
+      const { custom, ...messageData } = message;
+
+      expect(firebaseMessage.tokens).to.deep.equal(regIds);
+
+      expect(firebaseMessage.android.priority).to.equal('high');
+      expect(firebaseMessage.android.notification).to.deep.include(messageData);
+
+      expect(firebaseMessage.apns.payload.aps.sound).to.equal(
```

Comment:
This should be `firebaseMessage.android.payload.aps.sound`

Reply:
Not according to the types: https://github.com/firebase/firebase-admin-node/blob/a37eb6cb5ad92d911d0b868f5beb8d11761bcc0a/src/messaging/messaging-api.ts#L20

the `android?: AndroidConfig` does not have a `payload` property, but `apns?: ApnsConfig;` does, which has a `aps: Aps;` that contains a `sound?: string | CriticalSound;` property.